libs/sdk-ui-kit/src/Dialog/ShareDialog/types.ts

```diff
@@ -55,7 +56,63 @@ export interface IShareDialogProps {
     onApply: (payload: ISharingApplyPayload) => void;
     onCancel: () => void;
     onError?: (error: GoodDataSdkError) => void;
+    onInteraction?: (data: ShareDialogInteractionData) => void;
     isLockingSupported: boolean;
     labels: IShareDialogLabels;
     currentUserPermissions: CurrentUserPermissions;
 }
+
+// TODO: this will be fined tuned according to UX and PO requirements for action types
+/**
+ * @internal
+ */
+export type ShareDialogInteractionType =
+    | "DIALOG_OPENED"
```

Comment:
Is this key ? DIALOG_OPENED? This one is without prefix

Reply:
updated